modifytags/modifytags.go

```diff
@@ -0,0 +1,367 @@
+package modifytags
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"go/ast"
+	"go/token"
+	"sort"
+	"strconv"
+	"strings"
+	"unicode"
+
+	"github.com/fatih/camelcase"
+	"github.com/fatih/structtag"
+)
+
+// A Mutation defines how the input is modified.
```

Comment:
"A Mutation defines how struct tags should be modified for a given input struct." perhaps?

("the input" is a bit ambiguous)